### DISCUSSION TITLE
custom detection

### DISCUSSION THREAD

**Discussion by erkara** [mikel-brostrom/yolo_tracking#474] [General]
Hi all

Thank you for this great library. I wonder if your library can be modified (can handle) to accept a custom Pytorch model trained to detect a single class?
I would super help my research thank you!

**Comment by mikel-brostrom** [comment]
> I wonder if your library can be modified (can handle) to accept a custom Pytorch model trained to detect a single class?

Thir repo can load any Yolov5 model you have trained, for other architectures than Yolov5 you would need to customize the repo yourself

**Reply by erkara** [reply]
I am doing single class detection-tracking and that class is not one of COCO or others. Is there any tutorial for custom-datasets I can check out?

**Reply by mikel-brostrom** [reply]
https://github.com/ultralytics/yolov5/wiki/Train-Custom-Data

**Reply by apaaris** [reply]
In that case, admitting that we train to detect a new category which is not in COCO, which StrongSORT weights should be used?
I would have a pretrained model which works for my custom category and I was looking on how to implement tracking, since ideally it's possible to load the yolo weights directly, I was wondering if there are any adjustments to be made to the StrongSORT part

**Comment by mikel-brostrom** [comment]
You could train a custom ReID model on your specific class for better ReID performance.
